basic_pitch/predict.py

```diff
@@ -127,27 +128,34 @@ def main() -> None:
     for audio_path in audio_path_list:
         verify_input_path(audio_path)
 
-    predict_and_save(
-        audio_path_list,
-        output_dir,
-        args.save_midi,
-        args.sonify_midi,
-        args.save_model_outputs,
-        args.save_note_events,
-        pathlib.Path(args.model_path),
-        args.onset_threshold,
-        args.frame_threshold,
-        args.minimum_note_length,
-        args.minimum_frequency,
-        args.maximum_frequency,
-        args.multiple_pitch_bends,
-        not args.no_melodia,
-        pathlib.Path(args.debug_file) if args.debug_file else None,
-        args.sonification_samplerate,
-        args.midi_tempo,
-    )
-
-    print("\n✨ Done ✨\n")
+    try:
+        predict_and_save(
+            audio_path_list,
+            output_dir,
+            args.save_midi,
+            args.sonify_midi,
+            args.save_model_outputs,
+            args.save_note_events,
+            pathlib.Path(args.model_path),
+            args.onset_threshold,
+            args.frame_threshold,
+            args.minimum_note_length,
+            args.minimum_frequency,
+            args.maximum_frequency,
+            args.multiple_pitch_bends,
+            not args.no_melodia,
+            pathlib.Path(args.debug_file) if args.debug_file else None,
+            args.sonification_samplerate,
+            args.midi_tempo,
+        )
+        print("\n✨ Done ✨\n")
+    except IOError as ioe:
+        print(ioe)
+    except Exception as e:
+        print("🚨 Something went wrong 😔 - see the traceback below for details.")
```

Comment:
nice

Reply:
little hesitant here though because exception is so general. If something goes wrong that's not raised by our code, do we still want this behavior?

Reply:
Yeah I definitely agree that exception handling could be more specific! This logic was already existing, I'm just bubbling up the error from a [subclass](https://github.com/spotify/basic-pitch/pull/84/files#diff-e3aa4b8d1babc74d9aa581598275dbd5b17c96e15beff65edc110e56b6cc2e90L435) to main.